web/cypress/integration/pluginTemplate/create-edit-delete-plugin-template.spec.js

```diff
@@ -74,6 +75,17 @@ context('Create Configure and Delete PluginTemplate', () => {
 
     cy.contains('Submit').click();
     cy.contains('Next').click();
+    cy.contains(selector.pluginCard, 'basic-auth').should('be.visible');
+    cy.contains(selector.pluginTitle, 'Authentication').should('be.visible');
+    cy.contains(
+      selector.pluginTitle,
+      'Security',
+      'Traffic Control',
+      'Serverless',
+      'Observability',
+      'Other',
```

Comment:
I'm not sure this is the right way to write them, should we split them into 5 assertions